src/neuroglancer/datasource/precomputed/frontend.ts

```diff
@@ -112,6 +121,44 @@ export class MultiscaleVolumeChunkSource implements GenericMultiscaleVolumeChunk
   }
 }
 
+
+export class SkeletonSource extends BaseSkeletonSource {
+  get skeletonVertexCoordinatesInVoxels() {
+    return false;
+  }
+  get vertexAttributes() {
```

Comment:
the radius of vertex can be encoded in the attributes for future visualization?

Reply:
You can store arbitrary data in every vertex that can later be used for visualization